router/router.go

```diff
@@ -669,10 +669,28 @@ func (worker *workerT) processDestinationJobs() {
 								if worker.rt.transformerProxy {
 									jobID := destinationJob.JobMetadataArray[0].JobID
 									pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request started`, worker.rt.destName, jobID)
+
+									// setting metadata
+									firstJobMetadata := destinationJob.JobMetadataArray[0]
```

Comment:
Are these changes backward compatible with older transformer versions?

Reply:
Yes, for the older transformer, this would act as extra information being sent & this information would not be used
For the new transformer, this information would be useful

Reply:
Even if the server is of older version, the metadata information will not be sent. Currently handling is done appropriately to handle this case.